### PR TITLE
refactor: throwing of error when unable to save to db

### PR DESCRIPTION
**Pull request by sshanzel**
When the error is not about duplicated entries, we should throw an error.

### CODE REVIEW

**Review by TomDowling — APPROVED** [review]
Looks good to me

File: src/schema/comments.ts [hunk 1]
```diff
@@ -832,7 +832,7 @@ export const resolvers: IResolvers<any, Context> = {
         });
       } catch (err) {
         if (err?.code !== TypeOrmError.DUPLICATE_ENTRY) {
-          ctx.log.error({ err }, 'failed to save report to database');
```
Comment (rebelchris) [inline src/schema/comments.ts:835]: I think we still need the logs, don't think the error does that but might be wrong

Comment (capJavert) [reply src/schema/comments.ts:835]: It should be caught by `fastify.setErrorHandler` and respond with `500` https://github.com/dailydotdev/daily-api/blob/7c6d96af88937563dfadc28aaae33c21dd103651/src/index.ts#L43-L46

Comment (idoshamun) [reply src/schema/comments.ts:835]: Yap, Ante's right

**Review by idoshamun — APPROVED** [review]

**Review by rebelchris — APPROVED** [review]

**Pull request merged**